README.md

```diff
@@ -25,7 +25,8 @@ As with many open source packages, we use [GitHub](https://github.com/analogdevi
 | Operating System        | GitHub master status  | Version |  Primary Installer Package  | Alternative Package, tarball or zip |
 |:-----------------------:|:---------------------:|:-------:|:-------------------:|:--------------:|
 | Windows                 | [![Windows Status](https://ci.appveyor.com/api/projects/status/github/analogdevicesinc/libiio?svg=true)](https://ci.appveyor.com/project/analogdevicesinc/libiio/branch/master) | Windows 10<br />Windows 8.1<br />Windows 8<br />Windows 7 | [![Latest Windows installer](https://raw.githubusercontent.com/wiki/analogdevicesinc/libiio/img/win_box.png)](https://ci.appveyor.com/api/projects/analogdevicesinc/libiio/artifacts/libiio-setup.exe?branch=master) | [![Latest Windows zip](https://raw.githubusercontent.com/wiki/analogdevicesinc/libiio/img/win_box.png)](https://ci.appveyor.com/api/projects/analogdevicesinc/libiio/artifacts/libiio.zip?branch=master) |
-| OS X              |  [![OSX Status](https://api.travis-ci.org/analogdevicesinc/libiio.svg?branch=master&label=osx&passingTex=foo)](https://travis-ci.org/analogdevicesinc/libiio) |  OS X High Sierra <br />(v 10.13) | [![OS-X package 10.13](https://raw.githubusercontent.com/wiki/analogdevicesinc/libiio/img/osx_box.png)](http://swdownloads.analog.com/cse/travis_builds/master_latest_libiio-darwin-10.13.6.pkg) | [![OS-X tarball 10.10](https://raw.githubusercontent.com/wiki/analogdevicesinc/libiio/img/osx_box.png)](http://swdownloads.analog.com/cse/travis_builds/master_latest_libiio-darwin-10.13.6.tar.gz) |
+| OS X                    |  [![OSX Status](https://api.travis-ci.org/analogdevicesinc/libiio.svg?branch=master&label=osx&passingTex=foo)](https://travis-ci.org/analogdevicesinc/libiio) |  OS X High Sierra <br />(v 10.14) | [![OS-X package 10.14](https://raw.githubusercontent.com/wiki/analogdevicesinc/libiio/img/osx_box.png)](http://swdownloads.analog.com/cse/travis_builds/master_latest_libiio-darwin-10.14.4.pkg) | [![OS-X tarball 10.14](https://raw.githubusercontent.com/wiki/analogdevicesinc/libiio/img/osx_box.png)](http://swdownloads.analog.com/cse/travis_builds/master_latest_libiio-darwin-10.14.4.tar.gz) |
```

Comment:
I think that 10.14.4 is Mojave, not High Sierra - so the name needs an update. 

https://en.wikipedia.org/wiki/MacOS_Mojave

Reply:
updated;
thanks for catching this;
i think we would really need to see about a more dynamic update-thingi for this readme;
i'll try to think of something that scales;
maybe something that auto-generates from `.travis.yml`

Reply:
There isn't a way to codify the xcode version in travis-ci to the macos name; other than manually editing some file somewhere.

But I get your point. It's been on my list for awhile to have the ci system automatically generate in index.html file which points to everything in a flat list - but I don't think this would have the actual names (which is what most end users key off of)... I'm still noodling on that too...